nhsn/params.json.template

```diff
@@ -0,0 +1,22 @@
+{
+  "common": {
+    "export_dir": "./receiving",
+    "log_filename": "NAME.log"
+  },
+  "validation": {
+    "common": {
+      "data_source": "NAME",
+      "span_length": 14,
+      "min_expected_lag": {"all": "1"},
+      "max_expected_lag": {"all": "3"},
+      "dry_run": true,
+      "suppressed_errors": []
```

Comment:
Check on these [here](https://github.com/cmu-delphi/covidcast-indicators/blob/feea58d900cdb2ac4988d4cf1fb9ef96bbb54181/_delphi_utils_python/delphi_utils/validator/README.md)

Reply:
?

Reply:
Validation is triggered in production runs when there's a validation section in params.json like this and the runner is used to trigger the indicator run (`env/bin/python -m delphi_utils.runner delphi_${ind_name}` instead of `env/bin/python -m delphi_${ind_name}`).

You can remove the whole validation section if it's not needed for the indicator. But if you want to run the indicator with validation, the link has some more info on how to configure it in accordance with lag behaviors and such.